### PR TITLE
add eventConfig missing method

### PR DESCRIPTION
**Pull request by mowangjuanzi**
1. add `eventConfig::setFlags`. Source code reference: https://bitbucket.org/osmanov/pecl-event/src/1fa696a0dc145d5d5f40a144f7a27be31eaf14a5/php8/classes/event_config.c#lines-135
2. add `eventConfig::requireFeatures` missing return value description
3. add `EventConfig::setMaxDispatchInterval` required pecl event version

### CODE REVIEW

File: reference/event/eventconfig/requirefeatures.xml [hunk 1]
```diff
@@ -40,7 +40,9 @@
  </refsect1>
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
-  <para></para>
+  <para>
+   Returns &true; on success, otherwise &false;.
```
Comment (sy-records) [inline reference/event/eventconfig/requirefeatures.xml:44]: You can just use `&return.success;`

```suggestion
   &return.success;
```

File: reference/event/eventconfig/setflags.xml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  <para>
+   Returns &true; on success, otherwise &false;.
+  </para>
+ </refsect1>
+<refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Returns &true; on success. Otherwise &false;.
+  </para>
+ </refsect1>
```
Comment (sy-records) [inline reference/event/eventconfig/setflags.xml:54]: Repeat, the same can be done with `&return.success;`
```suggestion
<refsect1 role="returnvalues">
  &reftitle.returnvalues;
  <para>
   &return.success;
  </para>
 </refsect1>
```

**Review by sy-records — CHANGES_REQUESTED** [review]

File: reference/event/eventconfig/setflags.xml [hunk 3]
```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
+<refentry xml:id="eventconfig.requirefeatures" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (sy-records) [inline reference/event/eventconfig/setflags.xml:3]: You need to change the id.

```suggestion
<refentry xml:id="eventconfig.setflags" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
```

**Review by sy-records — CHANGES_REQUESTED** [review]

File: reference/event/eventconfig/requirefeatures.xml (hunk 1 above)
Comment (mowangjuanzi) [reply reference/event/eventconfig/requirefeatures.xml]: Is this a shorthand?

Comment (cmb69) [reply reference/event/eventconfig/requirefeatures.xml]: Yes; see https://github.com/php/doc-en/blob/668b6fc28891062b1c96202009ccb70b7147740d/language-snippets.ent#L648.

Comment (mowangjuanzi) [reply reference/event/eventconfig/requirefeatures.xml]: i see.

File: reference/event/eventconfig/setflags.xml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  <variablelist>
+   <varlistentry>
+    <term>
+     <parameter>flags</parameter>
+    </term>
+    <listitem>
+     <para>
+      One of <literal>EventBase::LOOP_*</literal>constants.
```
Comment (sy-records) [inline reference/event/eventconfig/setflags.xml:33]: Need a space 

```suggestion
      One of <literal>EventBase::LOOP_*</literal> constants. 
```

**Review by sy-records — CHANGES_REQUESTED** [review]

**Review by sy-records — APPROVED** [review]
Thank you.

**Pull request merged**